### PR TITLE
Add radar as trusted UID in run-gitmanager container settings

### PR DESCRIPTION
**Pull request by PasiSa**
# Description

**What?**

For some reason also aplus needed to be added to the UID set (even if whole attribute was not specified before), otherwise the messages from aplus were not authorized anymore (?)

**Why?**

[ANSWER HERE]

**How?**

[ANSWER HERE]

Fixes #<issue number>

# Testing

**Remember to add or update unit tests for new features and changes.**

* How to [test your changes in A-plus](https://github.com/apluslms/a-plus/tree/master/doc#running-tests-and-updating-translations)
* How to [test accessibility](https://wiki.aalto.fi/display/EDIT/How+to+check+the+accessibility+of+pull+requests)

**What type of test did you run?**

- [ ] Accessibility test using the [WAVE](https://wave.webaim.org/extension/) extension.
- [ ] Django unit tests.
- [ ] Selenium tests.
- [ ] Other test. *(Add a description below)*
- [ ] Manual testing.

[ADD A DESCRIPTION ABOUT WHAT YOU TESTED MANUALLY]

**Did you test the changes in**

- [ ] Chrome
- [ ] Firefox
- [ ] This pull request cannot be tested in the browser.

**Think of what is affected by these changes and could become broken**

# Translation

- [ ] Did you modify or add new strings in the user interface? ([Read about how to create translation](https://github.com/apluslms/a-plus/tree/master/doc#running-tests-and-updating-translations))

# Programming style

- [ ] Did you follow our [style guides](https://apluslms.github.io/contribute/styleguides/)?
- [ ] Did you use Python type hinting in all functions that you added or edited? ([type hints](https://docs.python.org/3/library/typing.html) for function parameters and return values)

# Have you updated the README or other relevant documentation?

- [ ] documents inside the doc directory.
- [ ] README.md.
- [ ] Aplus Manual.
- [ ] Other documentation (mention below which documentation).

# Is it Done?

- [ ] Reviewer has finished the code review
- [ ] After the review, the developer has made changes accordingly
- [ ] Customer/Teacher has accepted the implementation of the feature

*Clean up your git commit history before submitting the pull request!*

### CODE REVIEW

File: docker/rootfs/srv/gitmanager-cont-settings.py [hunk 1]
```diff
@@ -87,4 +87,25 @@
...
+MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAvddPdRsfeYjTK/aVnX/J
+52exmXiu7ZXLx2W83brLmxaOGR1Zb0qFt+0rtvBAx+s9KjlRj8rmMBLpTUTWWeai
+MpVHkhMH+MHAcL8jCfM8G5nLqcg2j50VBfXEiKT0QtonkOH+HVLYrtR0ZRQt4i9/
+8XLi1z+oITlH30yqs19PvZVksIJjReLqRDI1bxdzs6296i2Js5PyvGNKY1cn52dq
+MgMysg9P3HeuwAQW2jXQwxPn4HhNoKlQL2SpNvInWwmpS1PrXgIhEvEq+T79GcxI
+eGJ1Rjhi6HY9jhFMYBh23EirAa+HRBAbcyQ01Cc8hUa2YoNoolD/3oadsfQshps8
+8wIDAQAB
+-----END PUBLIC KEY-----""",
+     },
+     "TRUSTED_UIDS": ["radar", "aplus"],
```
Comment (Mankro) [inline docker/rootfs/srv/gitmanager-cont-settings.py:110]: It looks like aplus-auth adds `REMOTE_AUTHENTICATOR_UID` automatically to `TRUSTED_UIDS` **only if** `TRUSTED_UIDS` is not specified at all in the settings. Therefore, we have to include `"aplus"` in `TRUSTED_UIDS` here even though it feels a bit wrong.

https://github.com/apluslms/aplus-auth/blob/8050cecd99dd9335adf684a2ec57c068e7a67edd/src/aplus_auth/__init__.py#L132-L133

However, it looks like `UID_TO_KEY` should always automatically include `REMOTE_AUTHENTICATOR_KEY` and thus, it should not be necessary to copy `"aplus"` to `UID_TO_KEY`. It has already been defined in `REMOTE_AUTHENTICATOR_KEY` and `REMOTE_AUTHENTICATOR_UID`.

https://github.com/apluslms/aplus-auth/blob/8050cecd99dd9335adf684a2ec57c068e7a67edd/src/aplus_auth/__init__.py#L135-L138

**Review by Mankro — CHANGES_REQUESTED** [review]
Great! I think `aplus` can be removed from `UID_TO_KEY`, so could you check that?

**Comment by Mankro** [comment]
The radar key in the aplus-auth settings corresponds to the update of the run-radar container in https://github.com/apluslms/radar/pull/40

File: docker/rootfs/srv/gitmanager-cont-settings.py (hunk 1 above)
Comment (PasiSa) [reply docker/rootfs/srv/gitmanager-cont-settings.py:101]: You are correct. Removed aplus from UID_TO_KEY and force pushed.

**Review by Mankro — APPROVED** [review]

**Pull request merged**